app/src/main/java/xyz/malkki/neostumbler/ui/screens/settings/SettingsScreen.kt

```diff
@@ -50,6 +51,7 @@ fun SettingsScreen() {
             GeosubmitEndpointSettings()
             AutoUploadToggle()
             DbPruneSettings()
+            CoverageLayerSettings()
```

Comment:
I feel like this should be placed directly below the endpoint configuration

Reply:
I done it via https://github.com/mjaakko/NeoStumbler/compare/a6077a257e0d9492838691b6c82814f9a4629907..93d572b5d4ec354df5c1113b07c5300d79f21904